app/views/citizens/outstanding_mortgages/show.html.erb

```diff
@@ -5,5 +5,8 @@
 <% end %>
 
 <%= render partial: 'shared/forms/error_summary', locals: { model: @form, attribute_prefix: :legal_aid_application_ } %>
-
-<%= render partial: '/shared/forms/outstanding_mortgage_form', locals: { model: @form, url: citizens_outstanding_mortgage_path } %>
+  <div class="govuk-grid-row">
```

Comment:
Looks like this is isn't aligned quite right - there's some whitespace that doesn't need to be there, the whole <div> block can be left-aligned (the providers version is correct).